optimum/exporters/openvino/convert.py

```diff
@@ -381,6 +381,9 @@ def export_pytorch(
     logger.info(f"Using framework PyTorch: {torch.__version__}")
     output = Path(output)
 
+    # TODO: temporary solution but statefulness should be added to the export config earlier
+    config.stateful = stateful
```

Comment:
what is this WA needed?
Do we still normally support stateful and stateless models?